lib/AST/DiagnosticConsumer.cpp

```diff
@@ -118,6 +118,11 @@ FileSpecificDiagnosticConsumer::subconsumerForLocation(SourceManager &SM,
   if (loc.isInvalid())
     return None;
 
+  // What if a there's a consumer for fixits but there is no fixits output path?
```

Comment:
Nitpick: "fix-its". Also, I don't understand the point here—what makes fix-it consumers different from serializing consumers?

Reply:
Tx for the nitpick. I've clarified the comment, hope it works better now.